web-ng/src/app/components/card-view-project/card-view-project.component.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, OnDestroy, ElementRef, OnInit } from '@angular/core';
+
+import { ProjectService } from '../../services/project/project.service';
+import { Subscription } from 'rxjs';
+import { MatDialog } from '@angular/material/dialog';
+import { UserProfilePopupComponent } from '../../components/user-profile-popup/user-profile-popup.component';
+import { Project } from '../../shared/models/project.model';
+import { NavigationService } from '../../services/navigation/navigation.service';
+import { AuthService } from '../../services/auth/auth.service';
+
+@Component({
+  selector: 'app-card-view-project',
+  templateUrl: './card-view-project.component.html',
+  styleUrls: ['./card-view-project.component.scss'],
+})
+export class CardViewProjectComponent implements OnInit, OnDestroy {
+  projects?: Project[];
+  private subscription = new Subscription();
+  breakpoint: number;
```

Comment:
Can you please add comments explaining what this is an how it's used?

Reply:
removed was no longer required